### PR TITLE
feat: [CVS-9234]: delete all custom metric

### PR DESCRIPTION
**Pull request by deepesh-ui**
##### Summary:  

-  Go to AppD and after filling out tier and metric category information click on add a metric. If the user accidentally clicked on add a metric there is no way for them to remove that metric. User has to go back to connector screen and reset the entire experience.
- User should be able to delete all custom metric if already added and revert to view there is no custom metric 
 applicable for  AppDynamics and NewRelic

##### Jira Links:  

- https://harness.atlassian.net/browse/CVS-9234 
- https://harness.atlassian.net/browse/CVS-9242

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by deepesh-ui** [comment]
retrigger jest

**Comment by deepesh-ui** [comment]
retrigger standards

**Comment by deepesh-ui** [comment]
retrigger jest

**Comment by deepesh-ui** [comment]
trigger cypress

File: cypress/integration/85-cv/HealthSource/AppDynamics.spec.ts [hunk 1]
```diff
@@ -151,7 +152,7 @@ describe('Create empty monitored service', () => {
     cy.contains('span', 'Submit').click({ force: true })
 
     cy.contains('div', 'AppD').click({ force: true })
-    cy.contains('span', 'Next').click()
+    cy.contains('span', 'Next').click({})
```
Comment (praneshg239) [inline cypress/integration/85-cv/HealthSource/AppDynamics.spec.ts:155]: why this empty object?

**Review by praneshg239 — APPROVED** [review]

**Comment by praneshg239** [comment]
trigger cypress

**Review by vasuki-kalluri — APPROVED** [review]

**Pull request merged**